### PR TITLE
Us 37

### PR DESCRIPTION
**Pull request by jgrazulis**
## Description of Changes

- Added model and feature test for` items_not_shipped` method
- Added method to model
- Added test data
- Added method to `merchant_dashboard` index
-  Changed describe block in dashboard spec to 'merchant dashboard'

## Concerns (tags recommended)

- When I ran the test it showed one failure for us 39 as uninitialized constant 'InvoicesController'

## What gif describes how you feel about this PR?
the-goon-star-wars

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :merchants do
+    resources :invoices
```
Comment (glsorensen) [inline config/routes.rb:4]: This is fixed in the other PR with the routes, don't know if you want to leave it there

File: spec/features/merchants/dashboard_spec.rb [hunk 2]
```diff
@@ -25,4 +68,15 @@
       expect(current_path).to eq("/merchants/#{@merchant1.id}/items")
     end
   end
+
+  describe 'items ready to ship' do
+    it 'shows section for items ready to ship with names of items not yet shipped' do
+      expect(page).to have_content("Items Ready to Ship")
+      expect(page).to have_content(@invoice_1.created_at.strftime("%A, %B %d,%Y"))
```
Comment (glsorensen) [inline spec/features/merchants/dashboard_spec.rb:75]: Nice, greg and I were wondering how to do the date.

File: spec/models/merchant_spec.rb [hunk 3]
```diff
@@ -49,5 +54,9 @@
       results = [@customer_2, @customer_3, @customer_4, @customer_5, @customer_6]
       expect(@nike.top_five_customers).to eq(results)
     end
+
```
Comment (gregoryjflaherty) [inline spec/models/merchant_spec.rb:57]: Great testing here - looks good

File: spec/features/merchants/dashboard_spec.rb (hunk 2 above)
Comment (gregoryjflaherty) [reply spec/features/merchants/dashboard_spec.rb:75]: Yes, this way is very clean

File: config/routes.rb (hunk 1 above)
Comment (gregoryjflaherty) [reply config/routes.rb:4]: Is this where the failed test is coming from?

**Pull request merged**